src/constants/headerConstants.js

```diff
@@ -0,0 +1,9 @@
+const constants = {
```

Comment:
I ask you never to call a constant like that again))) And rename current constant objects

Reply:
Ok. Thank you. I will remember this